src/main.js

```diff
@@ -3,6 +3,14 @@ var coverImage = document.querySelector('.cover-image');
 var coverTitle = document.querySelector('.cover-title');
 var discriptorTagLine1 = document.querySelector('.tagline-1');
 var discriptorTagLine2 = document.querySelector('.tagline-2');
+var showNewCoverButton = document.querySelector('.random-cover-button');
+var makeOwnCoverButton = document.querySelector('.make-new-button');
+var mainPageDisplay = document.querySelector('.main-cover');
+var displayUserForm = document.querySelector('.form-view');
+var saveCoverButton = document.querySelector('.save-cover-button');
```

Comment:
Everything looks nice and cleaned up now. This was a great learning experience for us when it comes to naming conventions.